services/web/scripts/video/optimize-video

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Can we perhaps move these scripts into a `scripts/assets` folder (note that there are some naming inconsistencies `video/make-sprite` and `video/optimize-video` the latter should be `video/optimize`.

Would be good to create an overview `README.md` in `scripts/assets`

Reply:
Good idea, just added that.